src/main/java/mate/academy/onlinebookstore/entity/Category.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.onlinebookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
+@SQLDelete(sql = "UPDATE category SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
+@Table(name = "categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    private String description;
+
+    @Column(nullable = false, name = "is_deleted")
+    private boolean isDeleted = false;
+}
```

Comment:
Please keep one style for fields and lines between 
![image](https://github.com/loranazarenko/OnlineBookStore/assets/156338484/3ec92283-e207-4f0a-82cb-b841d70e6fdf)